### DISCUSSION TITLE
Little Confused | Help needed

### DISCUSSION THREAD

**Discussion by codezerro** [JdeRobot/RoboticsAcademy#2750] [Q&A] [answered]
I'm working on a new version of UI for Robotics Academy. One-third of my work is already finished. Should I share my work right now? So that other contributors can help, suggest improvements, or correct it.

Or share when finished. 

Its just demo screenshot

![Screenshot from 2024-09-28 21-41-45](https://github.com/user-attachments/assets/89089bc8-7d56-4077-a0d6-49f426689381)
![Screenshot from 2024-09-28 21-41-48](https://github.com/user-attachments/assets/963755fe-392a-4cc0-a149-a788d0e545cd)

**Comment by codezerro** [comment]
@jmplaza

**Reply by jmplaza** [reply, marked as answer]
Hi @codezerro , it looks nice!

Watch out, all RoboticsAcademy (RA) exercises are now based on ROS2.

Feel free to describe in an issue the problems UI you have detected or those you aim to solve with your new version. Sharing this will allow public discussion on your purpose and your proposed solution. The sooner you share it (videos, images, code, motivation...), the better. As maybe you could be solving a non interesting problem or maybe a better solution can be proposed from another community member.

Cheers!

**Reply by codezerro** [reply]
@jmplaza thank you.

**Reply by codezerro** [reply]
@jmplaza I'm working on it. I'll share my approach. I hope it benefits our community not only now but in the future as well.
